lametro/models.py

```diff
@@ -78,6 +78,29 @@ def get_last_action_date(self):
     def topics(self):
         return [s.subject for s in self.subjects.all()]
 
+    @property 
+    def is_viewable(self):
+        '''
+        Sometimes, a Bill may be imported to Councilmatic, though it should not be visible to the public.
+        This issue summarize why that might happen: https://github.com/datamade/la-metro-councilmatic/issues/345#issuecomment-421184826
+        Metro staff devised three checks for knowing when to hide or show a report:
+
+        (1) Is the view restricted, i.e., is `MatterRestrictViewViaWeb` set to True in the Legistar API? We skip these bills further upstream. https://github.com/opencivicdata/scrapers-us-municipal/pull/251
+
+        (2) Does the Bill have a classification of "Board Box"? Then, show it.
+
+        (3) Is the Bill on a published agenda, i.e., an event with the status of "passed" or "cancelled"? Then, show it.
+        '''
+        if self.bill_type == "Board Box":
+            return True
+
+        events_with_bill = Event.objects.filter(agenda_items__bill_id=self.ocd_id)
+        passed_events = [event for event in events_with_bill if (event.status == 'passed' or event.status == 'cancelled')]
```

Comment:
will this array ever become very large? since we only care about the presence, but not how many, etc., it may be more memory-efficient to do something like:

```python
for event in events_with_bill:
    if event.status in ('passed', 'cancelled'):
        return True
return False
```

Reply:
No, this list will never be very large –  a bill may be associated with one or two events. More than that does not align with what Metro has done historically.